pnpm-workspace.yaml

```diff
@@ -1,4 +1,5 @@
 packages:
+  - '.'
```

Comment:
Did you verify what you see when you run `pnpm change` after this modification?

Reply:
no but we can see what it will do in the changeset PR